charts/platform-system/templates/flux/_repositories/helm-repository-cert-manager.yaml

```diff
@@ -0,0 +1,10 @@
+{{ if and .Values.apps.certManager.enabled (eq .Values.solution.name "flux") }}
```

Comment:
filename: `helm-repository-jetstack.yaml` to match object name. Though if we want to name the object `cert-manager` and possibly have duplicate URLs that would be okay too.

Reply:
renamed to jetstack for consistency